testing/api/test_api.py

```diff
@@ -1444,3 +1446,26 @@ def test_delete_profile(testrun, reset_profiles, add_profile): # pylint: disable
   )
   # Check if profile was deleted
   assert deleted_profile is None
+
+
+def test_get_device_profile():
+  """ Test get device profile question"""
+  with open(DEVICE_PROFILE_QUESTIONS, "r", encoding="utf-8") as file:
+    questions = json.load(file)
+
+  # Checking the 400 error when a step does not exist.
```

Comment:
Should a 404 be returned for this instead of bad request if the steps are not found?

Reply:
> Should a 404 be returned for this instead of bad request if the steps are not found?

Done